library/std/src/env.rs

```diff
@@ -236,22 +236,18 @@ fn _var(key: &OsStr) -> Result<String, VarError> {
 }
 
 /// Fetches the environment variable `key` from the current process, returning
-/// [`None`] if the variable isn't set or there's another error.
+/// [`None`] in the following situations:
+///
+///  - the environment variable isn't set
+///  - the environment variable's name contains
+/// the equal sign character (`=`) or the NUL character
+///
+/// If this is not desired, consider using [`var_os`].
```

Comment:
```suggestion
```

Not sure what this line is supposed to convey. We already are at `var_os`, so why link to itself? Is it supposed to be `var`? If so, it has basically same behavior in regards to *not present*, `=` in value, `\0` in value.